src/apify/_actor.py

```diff
@@ -69,17 +70,31 @@ def __init__(
         self._configure_logging = configure_logging
         self._apify_client = self.new_client()
 
-        self._event_manager: EventManager
-        if self._configuration.is_at_home:
-            self._event_manager = PlatformEventManager(
+        # We need to keep both local & cloud storage clients because of the `force_cloud` option.
+        self._local_storage_client = MemoryStorageClient()
```

Comment:
Can't the local storage client be kept in the `service_locator`?

Reply:
In the service locator we keep the needed one - based on the run environemnt.

Reply:
And what is the benefit of that? Setting the storage client in the service locator based on the "is at home" seems OK to me.

Reply:
It gives the user the option to use whatever storage client they want when developing locally.